### DISCUSSION TITLE
Can we make a new release?

### DISCUSSION THREAD

**Discussion by ffMathy** [vitest-dev/vscode#236] [General]
New features have been merged to the main branch without releasing them. Can we push a new release?

**Comment by sheremet-va** [comment]
Please, [join Discord](https://discord.com/channels/917386801235247114/919890069081440256) to get updates about this extension

**Reply by ffMathy** [reply]
@sheremet-va the given link doesn't work. Do I need to be invited to the channel?

Also, I recently created quite a lot of PRs for this repo. If it needs help, I'd love to become an official contributor as well!

**Reply by sheremet-va** [reply]
Sorry, I copied the wrong url! This is the correct one: https://discord.gg/d22CcMGy
